routes.go

```diff
@@ -117,24 +126,33 @@ func upload(c *gin.Context) {
 func upload_view(c *gin.Context) {
 	c.Data(http.StatusOK, "text/html", []byte(`
 	<html>
-	<body>
-		<h2>Upload a Plug!</h2>
-		<p>You will lose 1 drink credit in exchange for a 100 view-limit plug!</p>
-		<p>Plugs must be 728x200 pixels and in PNG, or JPG format!</p>
-		<p>Due to High Volumes your Plug MUST be approved by a member of any of the following groups (drink, eboard, rtp)! Check /admin</p>
-		<div>
-			<form action="/upload" method="post" enctype="multipart/form-data">
-				<input type="file" name="file" id="file">
-				<input type="submit" value="Upload" name="submit">
-			</form>
-		</div>
-		<div>
-			<a href="https://github.com/liam-middlebrook/csh-plug">CSH: Plug on GitHub</a>
-		</div>
-		<script>
-alert("The CSH CodeOfConduct Section 8 prohibits the sending of content that may harass others. Please review the CSH CodeOfConduct before uploading content to Plug.");
-		</script>
-	</body>
+		<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">
```

Comment:
Even though you wouldn't necessarily pass in any template variables to this, I'd recommend moving it over to a template file so it's easier to read through `routes.go`